cmd/rosa-support/create/subnets/cmd.go

```diff
@@ -22,7 +22,7 @@ var Cmd = &cobra.Command{
 	Short: "Create subnets",
 	Long:  "Create subnets.",
 	Example: `  # Create a pair of subnets with prefix 'mysubnet-' in region 'us-east-2'
-  rosa-support create subnets --name-prefix=mysubnet --region us-east-2 --vpc-id <vpc id> --availability-zones <AZs>`,
```

Comment:
Just wondering, so there's no name-prefix param anymore?

Reply:
@chenz4027 this functionality was removed from the AWS client (which QE migrated from their own repo), now it is created with an automatically generated prefix within the client IIRC

Reply:
Got it!
/lgtm